src/sentry/deletions/defaults/group.py

```diff
@@ -25,34 +26,33 @@ def chunk(self):
         if self.last_event is not None:
             conditions.extend(
                 [
-                    ["timestamp", "<=", self.last_event.timestamp],
+                    ["timestamp", "<=", self.last_event["timestamp"]],
                     [
-                        ["timestamp", "<", self.last_event.timestamp],
-                        ["event_id", "<", self.last_event.event_id],
+                        ["timestamp", "<", self.last_event["timestamp"]],
+                        ["event_id", "<", self.last_event["event_id"]],
                     ],
                 ]
             )
 
-        events = eventstore.get_events(
-            filter=eventstore.Filter(
-                conditions=conditions, project_ids=[self.project_id], group_ids=[self.group_id]
-            ),
-            limit=self.DEFAULT_CHUNK_SIZE,
+        events = raw_query(
```

Comment:
An alternative approach to using `raw_query` could be to use `discover.query`. It is a high-level API that will let you select the subset of columns that you want.